docs/configuration.md

```diff
@@ -0,0 +1,53 @@
+# Configuration
+
+## Configuration methods
+
+## Global settings
```

Comment:
@MrAlias Let me know if there is a better way to structure the env vars into separate sections. I based these mostly on what I found in the .NET docs.

Reply:
lgtm